airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -708,6 +708,8 @@ class PullRequestStats(PullRequestSubstream):
     API docs: https://docs.github.com/en/rest/reference/pulls#get-a-pull-request
     """
 
+    state_checkpoint_interval = 100
```

Comment:
it seems we don't need this 
I see that this function 
https://github.com/airbytehq/airbyte/blob/f0cfaf6b35d45b99dcb2a9bed08f6d0bf8f2bfcb/airbyte-integrations/connectors/source-github/source_github/streams.py#L213
give the same result

Reply:
removed it as well